services/tenant-ui/package.json

```diff
@@ -14,12 +14,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.27.2",
     "body-parser": "^1.20.1",
     "config": "^3.3.8",
     "connect-history-api-fallback": "^2.0.0",
     "cors": "^2.8.5",
     "express": "^4.18.2",
+    "express-oauth-jwt": "^2.0.1",
     "http-proxy-middleware": "^2.0.6",
+    "jose": "^4.10.4",
```

Comment:
Curious as to what this library does.

Reply:
Jose is a JWT verification specification. Lots of libraries in lots of languages to support it. The Java one has a good explanation of where it comes from https://bitbucket.org/b_c/jose4j/wiki/Home